src/__tests__/__snapshots__/schema-base.test.ts.snap

```diff
@@ -307,6 +307,9 @@ input ItemCondition {
 
   \\"\\"\\"Checks for equality with the object’s \`createdAt\` field.\\"\\"\\"
   createdAt: Datetime
+
+  \\"\\"\\"The globally unique \`ID\` to be used in selecting a single \`Person\`.\\"\\"\\"
+  personByPersonOrganizationIdAndPersonIdentifier: [ID!]
```

Comment:
I'm still not comfortable with this naming for an array field, I'd prefer something like `personByPersonOrganizationIdAndPersonIdentifierIsOneOf` or `In` or `IsIn` or `ContainedIn`. 

Incidentally the description doesn't match (missing plural).